apps/wallet/src/ui/app/shared/card/index.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva, type VariantProps } from 'class-variance-authority';
+
+import type { ReactNode } from 'react';
+
+const cardContentStyle = cva([], {
+    variants: {
+        variant: {
+            blue: 'bg-sui/10',
+            white: 'bg-white',
+        },
+        padding: {
+            none: 'p-0',
+            small: 'p-3.5',
+        },
+    },
+    defaultVariants: {
+        variant: 'white',
+        padding: 'small',
+    },
+});
+
+export interface CardProps extends VariantProps<typeof cardContentStyle> {
+    header?: ReactNode;
+    footer?: ReactNode;
+    children?: ReactNode | ReactNode[];
+}
+
+export function Card({ header, footer, children, ...styleProps }: CardProps) {
+    return (
+        <div
+            className={
+                'rounded-2xl border border-solid border-gray-45 box-border overflow-hidden flex flex-col outline-1 w-full'
+            }
+        >
+            {header && (
+                <div className="bg-gray-40 flex justify-center items-center divide-x divide-solid divide-gray-45 divide-y-0">
```

Comment:
Not blocking at all but curious why divide is in the header

Reply:
You are right. I remove it.